.github/workflows/cd-v2-publish-and-create-pr.yml

```diff
@@ -0,0 +1,36 @@
+name: Publish prod version and create sync PR
+on:
+  push:
+    branches:
+      - v2
+
+jobs:
+  publish-and-create-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.node-version'
+          cache: 'yarn'
+          registry-url: 'https://registry.npmjs.org'
+      - name: Install dependencies
+        run: yarn --immutable
+      - name: Publish Production version
+        run: yarn changeset publish
```

Comment:
for first testing probably this should be commented out so we will check out if everything works as expected, wdyt @FRSgit  ?

Reply:
Yes, we can try it out - let's not add any rubbish on our npm by mistake 😄